lib/workers/repository/updates/generate.ts

```diff
@@ -466,6 +466,11 @@ export function generateBranchConfig(
     (upgrade) => upgrade.skipInstalls !== false,
   );
 
+  // Artifact updating will only be skipped if every upgrade wants to skip it.
```

Comment:
This should be added to the config option docs as a note, so users are awar of grouping behavior